Gopkg.lock

```diff
@@ -0,0 +1,628 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
The project has migrated to Go Modules and no longer uses dep. Please remove the gopkg files from the PR.

Reply:
Removed files